app/config/collections.php

```diff
@@ -2934,6 +2934,27 @@
                 'lengths' => [128],
                 'orders' => [Database::ORDER_ASC],
             ],
+            [
+                '$id' => ID::custom('_key_requestMethod'),
+                'type' => Database::INDEX_KEY,
+                'attributes' => ['requestMethod'],
+                'lengths' => [128],
+                'orders' => [Database::ORDER_ASC],
+            ],
+            [
+                '$id' => ID::custom('_key_requestPath'),
+                'type' => Database::INDEX_KEY,
+                'attributes' => ['requestPath'],
+                'lengths' => [Database::LENGTH_KEY],
```

Comment:
What is the max length of the request path? Will it be under 255 characters ?

Reply:
Good point. We use 2048 in the API, so changing it to that.